Riot/Modules/Room/Files/RoomFilesViewController.m

```diff
@@ -273,4 +308,71 @@ - (void)changeNavigationTitle {
     [[[[AppDelegate theDelegate] masterTabBarController] navigationItem] setTitle:NSLocalizedStringFromTable(@"tab_gallery", @"Vector", @"")];
 }
 
+#pragma mark - CameraPresenterDelegate
+
+- (void)cameraPresenterDidCancel:(CameraPresenter *)cameraPresenter
+{
+    [cameraPresenter dismissWithAnimated:YES completion:nil];
+    self.cameraPresenter = nil;
+}
+
+- (void)cameraPresenter:(CameraPresenter *)cameraPresenter didSelectImageData:(NSData *)imageData withUTI:(MXKUTI *)uti
+{
+    if (BuildSettings.sendMessageRequirePatientTagging){
+        
+        [cameraPresenter dismissWithAnimated:YES completion:^{
+            PatientTaggingViewController *PatientTaggingController = [PatientTaggingViewController new];
+            [PatientTaggingController setImageTo: imageData WithHandler:^(TagData *tagData) {
+                if (tagData.Patients.count > 0){
+                    //A patient was tagged
+                    //TODO: Replace this debug message with the actual actions necessary to tag the patient and upload to CPF.
+                    UIAlertController* alert = [UIAlertController alertControllerWithTitle:@"Tagged Photo Uploaded"
+                                                   message:@"The tagged photo of the patient would now be uploaded, with relevant tag data being stored by the CPF. This feature is pending on updates to the backend."
```

Comment:
Should this be externalised into the internationalised files? (NOTE: non-english translations are not required yet)

Reply:
Updated this controller to use string definitions for these test strings.  Only English strings were created.